custom/ewsghana/reports/__init__.py

```diff
@@ -231,6 +231,14 @@ class MultiReport(MonthWeekMixin, CustomProjectReport, CommtrackReportMixin, Pro
     is_exportable = False
     base_template = 'ewsghana/base_template.html'
 
+    def get_stock_transactions(self):
+        return StockTransaction.objects.filter(
+            case_id__in=list(SQLLocation.objects.get(
+                location_id=self.report_config['location_id']
+            ).get_descendants().exclude(supply_point_id__isnull=True).values_list('supply_point_id', flat=True)),
+            report__date__range=[self.report_config['startdate'], self.report_config['enddate']]
+        ).order_by('report__date', 'pk')
```

Comment:
This happens in several different places:
```python
SQLLocation.objects.get(location_id=self.report_config['location_id'])
```
I'd make a memoized property:
```python
@memoized
@property
def report_location(self):
    return SQLLocation.objects.get(location_id=self.report_config['location_id'])
```
This will help prevent the duplication and the extra db calls.